### PR TITLE
Replace kscript for the native cron report with jbang

### PR DESCRIPTION
**Pull request by geoand**
Also provide a deep link into the GH Action that run

This work was originally done for and tested with
https://github.com/quarkusio/quarkus-ecosystem-ci

### CODE REVIEW

File: .github/NativeBuildReport.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+			throw new UncheckedIOException(e);
+		}
+	}
+
+	private static boolean isOpen(GHIssue issue) {
+		return (issue.getState() == GHIssueState.OPEN);
+	}
+
+	// this example implements Callable, so parsing, error handling and handling user
+	// requests for usage help or version help can be done with one line of code.
```
Comment (gsmet) [inline .github/NativeBuildReport.java:92]: Minor nitpick: this comment looks useless?

Comment (geoand) [reply .github/NativeBuildReport.java]: Oooops, fixed :)

**Review by cescoffier — APPROVED** [review]

File: .github/NativeBuildReport.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.concurrent.TimeUnit;
+
+@Command(name = "report", mixinStandardHelpOptions = true,
+		description = "Takes care of updating the an issue depending on the status of the build")
```
Comment (machi1990) [inline .github/NativeBuildReport.java:18]:
```suggestion
		description = "Takes care of updating  an issue depending on the status of the build")
```

Comment (geoand) [reply .github/NativeBuildReport.java]: Updated, thanks

**Pull request merged**